src/Pages/ForgotPassword.cshtml.cs

```diff
@@ -46,7 +52,7 @@ public IActionResult OnGet()
                 return Page();
             }
             Email = Request.Cookies["Email"];
-            var recoveryCode = EmailSender.SendRecoveryCode(Email).ToString();
+            var recoveryCode = (await EmailSender.SendRecoveryCodeAsync(Email)).ToString();
```

Comment:
This seems nice improvement since it freezed when loading this page for a bit, did this fix it, if you have tested?

Reply:
I tested it out a bunch of times, and I can't say for sure if the performance has improved noticeably. However as I understand this is a must if multiple users would be using the website at once, because they couldn't all access the same static class at the same time, because smtp client is not thread safe. Could be wrong here though